ui/pages/confirmation/components/confirmation-network-switch/confirmation-network-switch.js

```diff
@@ -20,82 +18,54 @@ import {
 } from '../../../../../shared/constants/network';
 import { getProviderConfig } from '../../../../ducks/metamask/metamask';
 
-export default function ConfirmationNetworkSwitch({ newNetwork }) {
-  const { chainId, nickname, type } = useSelector(getProviderConfig);
-
+const NetworkIcon = ({ chainId, nickname }) => {
   return (
-    <Box
-      className="confirmation-network-switch"
-      display={Display.Flex}
-      height={BlockSize.Full}
-      justifyContent={JustifyContent.center}
-      marginTop={8}
-    >
-      <Box
-        className="confirmation-network-switch__icon"
-        display={Display.Block}
-      >
+    <Box className="confirmation-network-switch__icon-wrapper">
+      <Box className="confirmation-network-switch__icon">
         {chainId in CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP ? (
-          <SiteIcon
-            icon={CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP[chainId]}
+          <AvatarFavicon
+            margin="auto"
+            src={CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP[chainId]}
+            size={AvatarFaviconSize.Xl}
             name={nickname}
-            size={64}
           />
         ) : (
-          <div className="confirmation-network-switch__unknown-icon">
-            <i className="fa fa-question fa-2x" />
-          </div>
+          <i className="fa fa-question fa-2x" />
         )}
-        <Typography
-          color={TextColor.textDefault}
-          variant={TypographyVariant.H6}
-          fontWeight={FontWeight.Normal}
-          align={TextAlign.Center}
-          boxProps={{
-            display: Display.Flex,
-            justifyContent: JustifyContent.center,
-          }}
-        >
-          {nickname || NETWORK_TO_NAME_MAP[type]}
-        </Typography>
       </Box>
-      <Box
-        className="confirmation-network-switch__center-icon"
-        display={Display.Flex}
-        alignItems={AlignItems.center}
-        justifyContent={JustifyContent.center}
+      <Text
+        className="confirmation-network-switch__text"
+        color={TextColor.textDefault}
+        variant={TypographyVariant.H6}
```

Comment:
Check the `Text` component api the variant prop should use `TextVariant` but I think we could just use the default here
```suggestion
```